AppCreationScripts/Apps.json

```diff
@@ -35,7 +35,17 @@
             }
           ]
         }
-      ]
+      ],
+      "codeConfigurations": [	
+        {
+		      "settingFile": "/app_config.py", 
```

Comment:
Nit: If you prefer, you can remove those excess indentation, in order to match the existing indentation elsewhere in this file. This is technically optional, though. It is up to you.